### ISSUE TITLE
Museum donation

### ISSUE THREAD

**Issue by YaYeetBoyThe0P** [SkyblockerMod/Skyblocker#1071]
### Minecraft Version

1.21.0/1.21.1

### Skyblocker Version

1.21.0

### Reproduction Steps

1. Have an item you can donate in the museum 2. Donate it

### Crash Report file and latest.log

```shell
The game crashed whilst mouseclicked event handler
Error: java.lang.NullPointerException: Cannot invoke "it.unimi.dsi.fastutil.objects.ObjectOpenHashSet.add(Object)" because the return value of "de.hysky.skyblocker.skyblock.item.MuseumItemCache$ProfileMuseumData.collectedItemIds()" is null
```

**Comment by 7azeemm** [comment]
Can you send full log?
Was that your first donation ? (I mean empty museum?)

**Comment by kevinthegreat1** [comment]
Duplicate of #803, fixed in 9a59ee7. Please update. Also, older versions are no longer supported.

**Comment by YaYeetBoyThe0P** [comment]
It's still happening to me I'll try it on most recent version though see if that works
